tracer/src/Datadog.Trace/AspNet/TracingHttpModule.cs

```diff
@@ -172,23 +173,20 @@ private void OnBeginRequest(object sender, EventArgs eventArgs)
                 var security = Security.Instance;
                 if (security.Settings.Enabled)
                 {
-                    security.InstrumentationGateway.RaiseRequestStart(httpContext, httpContext.Request, scope.Span);
+                    SecurityCoordinator.ReportWafInitInfoOnce(security, scope.Span);
+                    var securityCoordinator = new SecurityCoordinator(security, httpContext, scope.Span);
```

Comment:
This call sequence seems a little odd to me (call a static method on the type, then create instance of the type using the same arguments) but I'll take your word for it that this makes the most sense 😅

Reply:
True, it's just at some other spot in Core, we just need this static method and dont need to create the instance 😅